test/postControllerSuperTest.js

```diff
@@ -0,0 +1,90 @@
+/* global it, describe */
+const express = require('express');
+const request = require('supertest');
+const postController = require('../controllers/postController');
+const mongoose = require('mongoose');
+const sinon = require('sinon');
+const postService = require('../services/postService');
+
+const app = express();
+
+mongoose.Promise = require('bluebird');
+
+mongoose.connect('mongodb://test:test@ds257627.mlab.com:57627/first-blog-mern', { useMongoClient: true });
```

Comment:
Your tests should not be using your actual database.
If you are writing an integration test, then you should be using a test database.
Otherwise, you should be mocking out your database